src/etls/bopz/scrapper.py

```diff
@@ -0,0 +1,162 @@
+import logging as lg
+import tempfile
+import typing as tp
+from datetime import date, datetime
+
+import re
+import requests
+from bs4 import BeautifulSoup
+from requests.exceptions import HTTPError
+from langchain_community.document_loaders import OnlinePDFLoader
+
+from src.etls.bopz.metadata import BOPZMetadataDocument
+from src.etls.common.scrapper import BaseScrapper
+from src.initialize import initialize_logging
+
+initialize_logging()
+
+# POST data to filter retrieved BOPZ documents
+data_post = {
```

Comment:
Use uppercase for global variable. Also, you can port this dict to a `utils.py` file.